### PR TITLE
Fix misspell documentation

### PR DESCRIPTION
**Pull request by aanm**
Running `make render-docs` can create the following output:
```
preparing documents... done
concepts.rst:143:orchestrator:pts                                                                                                                                                                                                                                                                                            
writing output... [100%] troubleshooting                                                                                                                                                                                                                                                                                     
Spelling checker messages written to /srv/Documentation/_build/spelling/output.txt
WARNING: Found 1 misspelled words
```
The build doesn't fail but if you think the word is not misspelled then add it to `Documentation/spelling_wordlist.txt`

### CODE REVIEW

**Comment by aanm** [comment]
test-me-please

**Comment by ianvernon** [comment]
Great addition of the spellchecker!

File: Documentation/spelling_wordlist.txt [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+backport
+backported
+backporting
+backports
+balancer
+barebones
+battlestation
+bazel
+bcc
+behaviour
```
Comment (ianvernon) [inline Documentation/spelling_wordlist.txt]: I'm surprised it doesn't all for non-American spelling of English words ;)

**Review by ianvernon — APPROVED** [review]

File: Documentation/spelling_wordlist.txt (hunk 1 above)
Comment (aanm) [reply Documentation/spelling_wordlist.txt]: Oh, since we are using en-US, "behaviour" is actually "wrong". It should be "behavior". I'll fix that one.

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

**Review by borkmann — APPROVED** [review]

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

**Pull request merged**